assignments/this.js

```diff
@@ -1,10 +1,10 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
```

Comment:
very good! short and concise answers that are correct and to the point. Love it.